### DISCUSSION TITLE
Amazon Echo Show Display mit Solectrus

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#2518] [General]
Die Echo Show Displays gibt es hin und wieder günstig angeboten.
Ist es möglich Solectrus dort anzuzeigen ?
Hat dies schon jemand erfolgreich umgesetzt ?
Wäre ne feine Sache. 
Die Displays haben eine gefällige Form.

**Comment by ledermann** [comment]
Von einem anderen User habe ich gehört, dass es auf einem "Echo Show 15" problemlos funktioniert.

Grundsätzlich ist es so, dass es nur auf den Browser ankommt. Der sollte möglichst modern sein, wobei die Anforderungen mit dem nächsten Update auf Version 0.14 nochmal etwas höher geschraubt werden.
Konkret heißt das: Chrome 106+, Safari 16+, Firefox 110+ (heikel wird es also, wenn der Browser älter als ein Jahr ist). Wohlgemerkt: Das Gerät kann älter sein, aber der Browser muss möglichst aktuell sein - was ja schon allein aus Sicherheitsgründen empfehlenswert ist.

Wenn ein konkreter älterer oder exotischer Browser Probleme macht, müsste ich im Einzelfall untersuchen, woran es hakt. Denkbar wäre dann, dafür eine spezielle Unterstützung zu bauen.

**Comment by StevieC121176** [comment]
Das funktioniert mit dem Amazon Silk Browser auch mit der aktuellen Solectrus Version. Ein anderer Browser wird zumindest bei den Show5 und 8 Geräten nicht unterstützt. Aufgrund der geringen Auflösung der Amazon Geräte wird aber nur die mobile Version im Landscape Modus angezeigt.

**Reply by ledermann** [reply]
Auf dem Show 15 habe ich schon Landscape-Screenshots gesehen. SOLECTRUS verwendet Landscape ab einer Breite von 1024px. Welche Auflösung haben denn die älteren Geräte?

**Reply by StevieC121176** [reply]
Soweit ich weiß in der aktuellen Revision auch nur 960x480 Pixel für den Echo Show 5 und 8

**Reply by ledermann** [reply]
Habe in der Wikipedia etwas gefunden: https://de.wikipedia.org/wiki/Amazon_Echo_Show#Vergleich
Demnach hat der Echo 5 960x480, aber der Echo 8 müsste 1280x800 bieten und somit Landscape verwenden.

**Reply by Zeppelin-PV** [reply]
Danke Euch, für Eure Hilfe zu diesem Thema.
Nebenbei habe ich herausgefunden dass diese allesamt sehr Strom hungrige Genossen sind.
Das schreckt mich von dem Vorhaben etwas ab.
Die Alexa Funktion benutze ich nicht. Es ging mir hauptsächlich um die Form des Displays.

**Reply by StevieC121176** [reply]
Ah ok, ich hab nur den Echo 5. Aber komfortabel ist der silk Browser nicht, besser ein altes Tablet nutzen
